packages/EasyErrorHandler/src/Response/ErrorResponseFactory.php

```diff
@@ -13,13 +13,15 @@
 
 final class ErrorResponseFactory implements ErrorResponseFactoryInterface, FormatAwareInterface
 {
+    private const PREFERRED_FORMAT = 'json';
```

Comment:
Shouldn't it be `SUPPORTED_FORMAT` instead?

Reply:
Thanks, done.